substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/events/JavaMonitorWaitEvent.java

```diff
@@ -39,7 +39,7 @@
 
 public class JavaMonitorWaitEvent {
     public static void emit(long startTicks, Object obj, long notifier, long timeout, boolean timedOut) {
-        if (HasJfrSupport.get()) {
+        if (HasJfrSupport.get() && !obj.getClass().getName().equals("jdk.jfr.internal.JVM$ChunkRotationMonitor")) {
```

Comment:
I'd prefer this to avoid NPEs:
```suggestion
        if (HasJfrSupport.get() && !"jdk.jfr.internal.JVM$ChunkRotationMonitor".equals(obj.getClass().getName())) {
```